src/components/PageHeader/PageHeader.stories.mdx

```diff
@@ -0,0 +1,78 @@
+import { Story, Canvas, ArgsTable } from '@storybook/addon-docs/blocks';
+import PageHeader from './PageHeader.jsx';
+
+<Meta title="PageHeader" component={PageHeader} />
+
+# PageHeader
+
+Very large text that can be used to present the title and short additional intro text for a non-article page.
+
+It takes optional children that should be used for relevant non-text components, such as navigation elements.
```

Comment:
We should probably just specify that this should always be a navigational element, e.g. tabs or a search bar. 'Non-text' components could be misleading

Reply:
Gotcha, I was looking around and it seemed like it should only be nav stuff but I wasn't sure if this was a hard rule